contract/models/contract.py

```diff
@@ -485,7 +485,10 @@ def _invoice_followers(self, invoices):
                 lambda x: invoice_create_subtype in x.subtype_ids
             ).mapped('partner_id')
             if partner_ids:
-                invoices.message_subscribe(partner_ids=partner_ids.ids)
+                item_invoices = item._get_related_invoices()
+                invoices.filtered(
+                    lambda x: item_invoices in x
```

Comment:
Wait, this is not correct, as `get_related_invoices` can return several old invoices:
```suggestion
                    lambda x: bool(item_invoices & x)
```

Reply:
I have fixed it in a more elegant way:
```python
(invoices & item._get_related_invoices()).message_subscribe(...
```